### ISSUE TITLE
Интересный тоглер

### ISSUE THREAD

**Issue by mihdan** [wpl-agency/dark-mode-for-elementor#1]
https://codepen.io/ste-vg/pen/oNgrYOb

**Comment by parfilov** [comment]
Да, прям красота. Получается бэкграундом по скрипту накрывает и на тогле прозрачность отрабатывает при назначении ID. Хм... Что то такое я и думал в плагин, вместо альфаканалов.
